computer.h

```diff
@@ -7,7 +7,7 @@
 class Computer: public Observer {
     ChessGame *subject;
     std::string name; // this is can acc probably be a str being either "Black" or "White"
-    std::vector<Piece> * pieceArray; //vector array of pieces for the computer
+    std::vector<Piece*> * pieceArray; //vector array of pieces for the computer
```

Comment:
We'll be adding a field `std::vector<Piece*> * oppArray;` that is the array of the pointers to the other player's pieces.